src/System.Net.Mail/tests/Unit/SmtpConnectionTests/EhloParseExtensionsTest.cs

```diff
@@ -113,7 +113,6 @@ public void ParseExtensions_WithNoMechanismsAdvertised_ShouldNotSupportAnyAuthTy
 
             Assert.True(_smtpConnection.DSNEnabled);
             Assert.True(_smtpConnection.ServerSupportsEai);
-            Assert.True(_smtpConnection.ServerSupportsStartTls);
```

Comment:
Why was this deleted?  It was useful as part of the unit test which tests the various internal properties.

Reply:
The property itself was deleted.

Reply:
Yes, I saw that also.  But wasn't the property valid?  And if so, was this then deleted because only the unit test was reading the property value?  And thus, no actual product code was using it?